src/helpers/utils.js

```diff
@@ -461,3 +463,20 @@ export const getSourcesCollections = (sources, getPathById) =>
       return acc;
     }, {})).filter(source => source && source.children && source.children.length);
 
+// helper function to filter array and get unique values - arr.filter(distinct)
+export const distinct = (value, index, arr) => arr.indexOf(value) === index;
```

Comment:
Don't use this this is not efficient, see comment above.

Reply:
removed

Reply:
I see the old code here, I don't see you have removed it. Did you update the PR?

Reply:
https://github.com/Bnei-Baruch/kmedia-mdb/pull/985/files#diff-6f75c83f4834bc3c6d394b362d541df2e842ff1ab52b1112a2a3aae9541498b9

Reply:
[https://github.com/Bnei-Baruch/kmedia-mdb/pull/985/files#diff-6f75c83f4834bc3c6d394b362d541df2e842ff1ab52b1112a2a3aae9541498b9](changes of utils)